pkgs/by-name/fi/filesender/package.nix

```diff
@@ -0,0 +1,43 @@
+{
+  stdenv,
+  fetchFromGitHub,
+  lib,
+  nixosTests,
+}:
+stdenv.mkDerivation (finalAttrs: {
+  pname = "filesender";
+  version = "2.48";
+
+  src = fetchFromGitHub {
```

Comment:
This project is a PHP project with a `composer.json` file containing dependencies. Where in this derivation do you resolve those dependencies?

Reply:
Hm. [FileSender installation guide](https://docs.filesender.org/filesender/v2.0/install/) doesn't mention composer installation.

> You can get the source either by [downloading a release](https://github.com/filesender/filesender/releases) or by using git. In both cases you should end up with a directory /opt/filesender containing the installation. The /opt/filesender directory will contain a filesender subdirectory with the FileSender code in it and a simplesaml subdirectory containing the authentication library used by FileSender.

Reply:
Fair enough. I'm so astonished to see this. How come is this even possible ?!

Reply:
Looks like it's only for development and tests. I'm not sure tho.

Reply:
Then it should be in `require-dev`, not in `require` !

Reply:
(and it works)